core/node/rules/can_add_event.go

```diff
@@ -204,7 +204,7 @@ func (params *aeParams) canAddChannelPayload(payload *StreamEvent_ChannelPayload
 	case *ChannelPayload_Redaction_:
 		return aeBuilder().
 			check(params.creatorIsMember).
-			requireChainAuth(params.redactChannelMessageEntitlements)
```

Comment:
a sendChannelMessage_Redaction does not result in a ChannelPayload_Redaction_, it's a ChannelPayload_Message. `redactMessage` results in a ChannelPayload_Redaction_, does this change your assumptions about this PR?

Reply:
Ah, I made a mistake here. My original PR sent a redaction and I fixed it to send a message with a redaction payload and just added a bug.

Reply:
Thank you for the edifying comment!

Reply:
I believe the channel message with redaction payload type is the right one here... We discussed it a while ago - this is the self-redaction option, does not require special permissions. and redactMessage is an admin-only client method.